script.js

```diff
@@ -1,12 +1,62 @@
+// hamburger icon /////////////////////////////////////////
 const hamburger = document.querySelector('.hamburger-icon');
```

Comment:
- [OPtional] please make sure that your popup is responsive, make sure that the width and height of the popup fits all the sizes
![image](https://user-images.githubusercontent.com/74744787/224182050-ca9b8709-44fb-414c-b64d-346e37b6d4ab.png)